articles/native-platforms/xamarin.md

```diff
@@ -44,22 +43,7 @@ There are three options to do the integration:
 
 To start with, we'd recommend using the __Login Widget__. Here is a snippet of code to copy & paste on your project:
 
-```cs
-using Auth0.SDK;
-
-var auth0 = new Auth0Client(
-	"@@account.namespace@@",
-	"@@account.clientId@@");
-
-// 'this' could be a Context object (Android) or UIViewController, UIView, UIBarButtonItem (iOS)
-var user = await auth0.LoginAsync(this);
-/*
-- get user email => user.Profile["email"].ToString()
-- get facebook/google/twitter/etc access token => user.Profile["identities"][0]["access_token"]
-- get Windows Azure AD groups => user.Profile["groups"]
-- etc.
-*/
-```
+@@snippet(meta.snippets.use)@@
```

Comment:
Let's separate this into 2 snippets, one on top of the other. The first 3 lines are the setup. Then it's the use. Makes sense? @ntotten